.github/workflows/dapr.yml

```diff
@@ -400,27 +303,30 @@ jobs:
           make release-flavor
         shell: bash
       - name: upload artifacts
-        uses: actions/upload-artifact@master
+        uses: actions/upload-artifact@v4
+        # Avoid publishing duplicate Windows artifacts, which will cause an error
+        if: matrix.windows_version != '1809'
```

Comment:
This is key to fixing the builds right now: https://github.com/dapr/dapr/actions/runs/7534402274/job/20516204786

(The binaries are the same on 1809 and LTSC2022)